go.mod

```diff
@@ -1,3 +1,5 @@
-module github.com/allegro/bigcache/v2
+module bigcache
```

Comment:
you shouldn't change this file 😄

Reply:
oh sorry, I accidentally pushed local changes to my master,